sysid-application/src/main/native/cpp/view/Generator.cpp

```diff
@@ -346,6 +356,15 @@ void Generator::Display() {
     ImGui::SameLine();
     ImGui::Checkbox(drive ? "L Inverted" : "Inverted", &pi[i]);
 
+    if (m_settings.motorControllers[i] == sysid::motorcontroller::kTalonFX ||
+        m_settings.motorControllers[i] == sysid::motorcontroller::kTalonFXPro) {
+      // Add CANivore name if we are using a CTRE motor controller
+      ImGui::SameLine();
+      ImGui::SetNextItemWidth(80);
+      ImGui::InputText("MC CANivore Name", m_settings.canivoreNames[i].data(),
+                       m_settings.canivoreNames[i].size());
```

Comment:
Not too familiar with the CANivore, but is it specific to only the Falcons?

Reply:
CANivore is specific to Falcons, CANcoder, and Pigeon2, which is why I have a field for all three of those devices.
https://github.com/wpilibsuite/sysid/pull/455/files#diff-4db26c65af78f49cb0a7385146e99aa2e25dcc97f6c52196d72c8bd8d5dff015R518
https://github.com/wpilibsuite/sysid/pull/455/files#diff-4db26c65af78f49cb0a7385146e99aa2e25dcc97f6c52196d72c8bd8d5dff015R148

It appears if I have multiple fields with the same name in ImGui, they tie to the same value. So I had to make their names unique since in concept you can have multiple CANivores on your robot, or at least separate devices between the CANivore bus and the RIO bus.
That said, I made the assumption that drivetrain Falcons are on the same bus, whether that is RIO or CANivore, which I think is a safe assumption since it seems odd to have half your drivetrain on one bus and the other half on another bus.
I'm also not tied to the current names in ImGui, so if you have suggestions on what to name the fields I'm all ears!